tests/D2L.CodeStyle.Analyzers.Test/Specs/ExplicitImmutableAttributeTransitivityAnalyzer.cs

```diff
@@ -84,4 +84,20 @@ public partial class
 
 	// This shouldn't crash the analyzer
 	public sealed class Foo : IThingThatDoesntExist { }
+

```

Comment:
Value in having a test case for the constructor style?

```public record ConciseRecord( string Blah );```

Reply:
Oh funky. Yes.

Reply:
It was good to explore where exactly the diagnostics landed with this syntax.